CMakeLists.txt

```diff
@@ -586,40 +588,44 @@ endif()
 
 
 # zlib
-if (NOT NACL)
-    find_package(ZLIB REQUIRED)
-else()
-    add_library(srclibs-zlib EXCLUDE_FROM_ALL ${ZLIBLIST})
-    set_target_properties(srclibs-zlib PROPERTIES POSITION_INDEPENDENT_CODE 1 FOLDER "libs")
-    set(ZLIB_INCLUDE_DIRS ${LIB_DIR}/zlib)
-    set(ZLIB_LIBRARIES srclibs-zlib)
+if (BUILD_CLIENT OR BUILD_TTY_CLIENT OR BUILD_SERVER OR NACL)
+    if (NOT NACL)
+        find_package(ZLIB REQUIRED)
+    else()
+        add_library(srclibs-zlib EXCLUDE_FROM_ALL ${ZLIBLIST})
+        set_target_properties(srclibs-zlib PROPERTIES POSITION_INDEPENDENT_CODE 1 FOLDER "libs")
+        set(ZLIB_INCLUDE_DIRS ${LIB_DIR}/zlib)
+        set(ZLIB_LIBRARIES srclibs-zlib)
+    endif()
+    set(LIBS_BASE ${LIBS_BASE} ${ZLIB_LIBRARIES})
+    include_directories(${ZLIB_INCLUDE_DIRS})
 endif()
-set(LIBS_BASE ${LIBS_BASE} ${ZLIB_LIBRARIES})
-include_directories(${ZLIB_INCLUDE_DIRS})
 
 if (DAEMON_PARENT_SCOPE_DIR)
     set(LIBS_BASE ${LIBS_BASE} PARENT_SCOPE)
 endif()
 
 # Curses, pdcurses on Windows and ncursesw on Unix
-if (USE_CURSES AND NOT NACL)
-    if (WIN32)
-        set(LIBS_ENGINE_BASE ${LIBS_ENGINE_BASE} gdi32 comdlg32)
-
-        add_definitions(-DPDC_WIDE -DPDC_FORCE_UTF8 -DUSE_CURSES)
-        add_library(srclibs-pdcurses EXCLUDE_FROM_ALL ${PDCURSESLIST})
-        if (MSVC)
-            target_compile_options(srclibs-pdcurses PRIVATE "/W2") # decrease warning level
+if (BUILD_CLIENT OR BUILD_TTY_CLIENT OR BUILD_SERVER OR BUILD_DUMMY_AP)
```

Comment:
BUILD_DUMMY_APP is spelled wrong and also there are 2 other cases where it is missing.

Reply:
oh lol, I'll fix that. Where are the other places? I don't know at all what needs or doesn't need the dummy app.

Reply:
See:

- https://github.com/DaemonEngine/Daemon/pull/754